src/Ovp/Twig/OvpExtension.php

```diff
@@ -22,49 +19,7 @@ public function __construct(ArtifactUrlProvider $artifactUrlProvider)
     public function getFunctions(): array
     {
         return [
-            new TwigFunction('ovp_artifact_url', [$this, 'getUrl']),
+            new TwigFunction('ovp_artifact_url', [ArtifactUrlProvider::class, 'getArtifactUrl']),
```

Comment:
@efjacobson @digitalartcrew to get around the static method problem, just keep the getUrl function and wiring and call the $this->artifactUrlProvider->getArtifactUrl(...) method in the getUrl method.